README – Linear Regression.md

```diff
@@ -0,0 +1,113 @@
+# Electric Car Battery Linear Regression Example
+
+### Linear Regression
```

Comment:
Can we shorten this section? Also, let's make it optional. e.g we can say something like this : here we have a short explanation of linear regression, feel free to skip it if you are already familiar with this.